rest-client/client-common/src/main/java/io/apicurio/registry/rest/client/exception/ExceptionMapper.java

```diff
@@ -59,6 +59,20 @@ public static RestClientException map(RestClientException ex) throws RestClientE
                 return new TenantNotFoundException(ex.getError());
             case "TenantNotAuthorizedException":
                 return new TenantNotAuthorizedException(ex.getError());
+            case "ContentNotFoundException":
+                return new ContentNotFoundException(ex.getError());
+            case "InvalidGroupIdException":
+                return new InvalidGroupIdException(ex.getError());
+            case "MissingRequiredParameterException":
+                return new MissingRequiredParameterException(ex.getError());
+            case "LogConfigurationNotFoundException":
+                return new LogConfigurationNotFoundException(ex.getError());
+            case "GroupNotFoundException":
+                return new GroupNotFoundException(ex.getError());
+            case "TenantManagerClientException":
+                return new TenantManagerClientException(ex.getError());
```

Comment:
I'm not sure if we want to propagate this to the rest layer. But since this happens once authenticated I guess we should be fine. Thoughts @famartinrh?